framework/src/main/java/org/tron/core/net/TronNetDelegate.java

```diff
@@ -365,4 +371,13 @@ public long getMaintenanceTimeInterval() {
     return chainBaseManager.getDynamicPropertiesStore().getMaintenanceTimeInterval();
   }
 
+  public boolean unsolidifiedBlockCheck() {
+    if (!unsolidifiedBlockCheck) {
+      return false;
+    }
+    long headNum = chainBaseManager.getHeadBlockNum();
+    long solidNum = chainBaseManager.getSolidBlockId().getNum();
+    return headNum - solidNum >= maxUnsolidifiedBlocks;
+  }
+
```

Comment:
Please rename the method unsolidifiedBlockCheck. It should not be the same as varible, it is confusing.

Reply:
well, done!